tv.kodi.Kodi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
+    "command": "kodi",
+    "rename-desktop-file": "kodi.desktop",
+    "rename-icon": "kodi",
+    "modules": [
+        "shared-modules/udev/udev-175.json",
+        "shared-modules/python2.7/python-2.7.15.json",
+        "deps/avahi.json",
+        "deps/libical.json",
+        "deps/bluez.json",
+        "deps/fmt.json",
+        "deps/fribidi.json",
+        "deps/fstrcmp.json",
+        "deps/ghostscript.json",
```

Comment:
Remove ghostscript. It isn't used/needed.

Reply:
right, i think it was the old runtime that made it need it, or an old fmt or fstrcmp version, but it compiles without it now